PersonApi.Tests/V1/Boundary/UpdatePersonRequestObjectTests.cs

```diff
@@ -1,35 +1,14 @@
 using AutoFixture;
 using FluentAssertions;
 using PersonApi.V1.Boundary.Request;
+using PersonApi.V1.Factories;
 using System;
 using Xunit;
 
 namespace PersonApi.Tests.V1.Boundary
 {
     public class UpdatePersonRequestObjectTests
     {
-        [Fact]
-        public void ToDatabaseTestEmptyGuidCreatesNewGuid()
-        {
-            var result = (new UpdatePersonRequestObject()).ToDatabase();
-            result.Id.Should().NotBe(Guid.Empty);
-        }
-
-        [Fact]
-        public void ToDatabaseTestNullSubObjectsCreatesDefaults()
-        {
-            var result = (new UpdatePersonRequestObject()).ToDatabase();
-            result.Identifications.Should().NotBeNull()
-                                       .And.BeEmpty();
-            result.Languages.Should().NotBeNull()
-                                 .And.BeEmpty();
-            result.CommunicationRequirements.Should().NotBeNull()
-                                        .And.BeEmpty();
-            result.PersonTypes.Should().NotBeNull()
-                                   .And.BeEmpty();
-            result.Tenures.Should().NotBeNull()
-                               .And.BeEmpty();
-        }
```

Comment:
Shouldn't this test just be changed to be something like `ToDatabaseTestNullSubObjectsCreatesNull()`?
And then each check can be something like `result.Identifications.Should().BeNull();`?